src/qat/purr/compiler/instructions.py

```diff
@@ -683,3 +688,45 @@ def __init__(self, variable: str, res_processing: InlineResultsProcessing):
 
     def __repr__(self):
         return f"{self.variable}: {str(self.results_processing.name)}"
+
+
+def remove_floating_point(x):
```

Comment:
inline this function please, it's too short and I don't think it's used anywhere else

Reply:
I get your point! But I'm planning to use it in Whisqrs for ArbSeq! Just to make the removal of floating points consistent and the same as QAT.

Reply:
the only thing worth sharing here is the rounding precision, which can be set properly via compiler config

cc/ @bgsach

Reply:
It's okay for now, assume this function is gonna disappear soon, and that you need to pick the rounding precision from the config instead, I don't want to see apis leaking this way between qat and internal dependencies

Reply:
Alright good to know theres a better solution going forward.